docs/osf-trivia-list/dasharo.md

```diff
@@ -235,6 +235,81 @@ Dasharo Tools Suite
 * We provide ready to use binaries with GPG based signing scheme that improve
   verification where firmware coming from.
 
+## What are the differences between the official coreboot repository and the Dasharo repository?
+
+* Dasharo focus on specific platforms listed in  [supported
+  hardware](https://docs.dasharo.com/unified/novacustom/overview/) section of Dasharo
+  Universe documentation.
+
+Dasharo repository contains release tags which are associated with Dasharo
+Certification Program providing certain quality criteria including test
+results. We always trying to minimize delta, but sometimes it can be up to 5k
+SLOC (or more I guess e.g. Talos II coreboot support).
+
+## What is Dasharo Certification Program?
+
+Dasharo Certification Program (DCP) is a rigorous certification program that
+sets a high-quality bar for open-source firmware. It aims to provide stable,
+secure, and reliable firmware that meets the Dasharo values. The program
+encourages developers to create their forks of Dasharo or contribute back to
+the Dasharo project or coreboot upstream. By doing so, they can leverage the
+power of open-source development to create custom firmware tailored to their
+specific needs. The certification process ensures that the firmware binary
+meets the strict criteria by testing  3mdeb in the certification lab.
+
+The Dasharo-certified firmware provides long-term maintenance over ten years
+after the CPU microarchitecture release, which means that OEM, ODM, hardware
+vendors, and other companies can rely on the firmware for a long time without
+worrying about end-of-life issues. Moreover, DCP-approved firmware vendors must
+provide professional support channels to ensure that other business entities
+can rely on those channels for long-term support regarding firmware and
+software.
+
+The validation process for DCP firmware is transparent. Test results and bug
+reports are always publicly available, allowing the community to continually
+identify issues and improve the firmware. However, in case of a security
+embargo, the results can be kept under a strict but well-defined policy,
+ensuring the security of the firmware.
+
+DCP-approved firmware respects privacy by allowing the deactivation of
+potentially malicious components, like ME or PSP, to the level permitted by the
+current state of technology. This feature is essential, especially in
+increasing data privacy concerns. DCP-approved firmware also respects the
+liberty of hardware owners to repair and transfer ownership without risking any
+personally identifiable data.
+
+DCP-approved firmware should provide trustworthiness. It means that known good
+measurements of each boot phase are publicly available and stored in a
```

Comment:
Do we have a plan / issue for that to happen?

Reply:
@macpijan technically I don't know what it involves, I'm trying to define some vision of the product. Getting to that vision need management and technical work which should happen on Dasharo product meetings. Please let me know if you expect any actions from me here.

Reply:
@pietrushnic 

My main problem with the threads which are still left open, is that (as you said) we are trying to define `some vision of the product`, but not the actual product (already available). With such description, no Dasharo firmware we release currently is DCP-approved. Is this what we want to say here in FAQ? This FAQ will be often linked and can be one of the first places users face. This may raise expectaions on the deliverables, which are not there, or maybe not even on the roadmap (even for releases supported via DSP, not only community ones).

My feeling is that the FAQ should rather refer to what is available, and when something is not, it should be clearly stated.

Maybe we need to define the criteria for `DCP-approved firmware` in a more organized manner (a table?), see which currently supported releases can score each requirment (and which not), define some set of requirements of the `DCP-approved firmware v1` and build a roadmap for `DCP-approved firmware v2` (or maybe v3, v4, etc) w .This should also encourage us to fill the missing features in the existing releases, and maybe also extend the new DSP contracts that we require that from certain date, supported firmware adheres to `DCP-approved list vX` and (given the current state of the platform) we need to implement following features.

Reply:
> My main problem with the threads which are still left open, is that (as you said) we are trying to define some vision of the product, but not the actual product (already available). With such description, no Dasharo firmware we release currently is DCP-approved.

It would be great to have such comment before we invest in improving content, that would not be approved. And I totally agree with your point.

DCP is to some extent WIP idea, but we already use it in practice but without correct marketing. Scope of DCP describe test matrix. At this point it is totally unstructured random set of tests associated, but having tiers in DCP would precisely explain scope that test matrix for given platform should apply. More to that we can over time advance given platform releases to higher tier DCP. I think we already will end up with that for Qubes OS hardware certification grades.

>  This FAQ will be often linked and can be one of the first places users face. This may raise expectaions on the deliverables, which are not there, or maybe not even on the roadmap (even for releases supported via DSP, not only community ones).

Agree and I'm ok to work more on this concept before it would be merged.

> My feeling is that the FAQ should rather refer to what is available, and when something is not, it should be clearly stated.

When filling this FAQ it happen to me to be natural to generate new ideas and product vision, how and where we should gather those concepts? Maybe there is better way than dropping that stuff here.

> Maybe we need to define the criteria for DCP-approved firmware in a more organized manner (a table?), see which currently supported releases can score each requirment (and which not), define some set of requirements of the DCP-approved firmware v1 and build a roadmap for DCP-approved firmware v2 (or maybe v3, v4, etc) w .This should also encourage us to fill the missing features in the existing releases, and maybe also extend the new DSP contracts that we require that from certain date, supported firmware adheres to DCP-approved list vX and (given the current state of the platform) we need to implement following features.

I'm probably way better in generating new ideas then in structuring those, but I will try. I'm worried that because of this new effort we have to freeze this part of FAQ in limbo for long time. Can we agree on what is acceptable to be merged in and we should remove? Maybe during the review you could just mark what is in your opinion new concept and should go elsewhere or wait for structured plan and what can be included since it describe today reality?

Reply:
> When filling this FAQ it happen to me to be natural to generate new ideas and product vision, how and where we should gather those concepts? Maybe there is better way than dropping that stuff here.

- FAQ - describe the actual state, or if describing vision which is not there, be very specific about mentioning that
- vision and new ideas can go to issues and MRs; we can even link some issues in the description to track their progress, if we mention a new idea in text and say it is planned, but bot there

That would be my view.

> Can we agree on what is acceptable to be merged in and we should remove? Maybe during the review you could just mark what is in your opinion new concept and should go elsewhere or wait for structured plan and what can be included since it describe today reality?

I marked two things with #future, which are not there and would prevent current releases from being called DCP-approved.